newtypesystem

```diff
@@ -0,0 +1,12 @@
+**Added:** None
+added mapstringmapstringdouble typesystem. This can be used with C++ archetypes. But not python archetypes. 
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:** None
+
+**Security:** None
```

Comment:
@gwenchee I did not notice this before...
could you actually place this file in news folder ?
and name it something like `typesystem.rst` ?

Reply:
@bam241 done! Sorry that was what I intended to do.